### ISSUE TITLE
React Version above 16.3.0 is needed

### ISSUE THREAD

**Issue by Tanner-sun** [HOUCe/react-quicklink-component#1]
Caution that :
> use the React.createRef() API to Creating Refs is introduced in React 16.3

With lower 16.3.0, use react-quicklink-component you will get an error
`Uncaught Error: Expected ref to be a function or a string.`

**Comment by HOUCe** [comment]
Yeah absolutely, i will support lower React version and PR also welcomes
